cmd/beaker/cluster.go

```diff
@@ -131,6 +134,37 @@ func newClusterCreateCommand() *cobra.Command {
 	return cmd
 }
 
+func newClusterExecutionsCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "executions <cluster>",
+		Short: "List executions in a cluster",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			executions, err := beaker.Cluster(args[0]).ListExecutions(ctx, nil)
+			if err != nil {
+				return err
+			}
+			return printExecutions(executions)
+		},
+	}
+}
+
+func executionStatus(state api.ExecutionState) string {
+	if state.Scheduled == nil {
+		return "pending"
+	}
+	if state.Started == nil {
+		return "starting"
+	}
+	if state.Ended == nil {
+		return "running"
+	}
+	if state.Finalized == nil {
+		return "finalizing"
+	}
+	return "finished"
```

Comment:
Style: this is a perfect time for a switch.
```suggestion
	switch {
	case state.Scheduled == nil:
		return "pending"
	case state.Started == nil:
		return "starting"
	case state.Ended == nil:
		return "running"
	case state.Finalized == nil:
		return "finalizing"
	default:
		return "finished"
	}
```